pkg/file_explorer/file_explorer.go

```diff
@@ -178,25 +162,10 @@ func (f *FileExplorer) updateEntries() {
 	}
 
 	l := len(*f.entries) - 1
-	if f.selected > l {
-		f.selected = l
-	}
-	if f.selected < 0 {
-		f.selected = 0
-	}
-}
-
-func repeatingKeyPressed(key ebiten.Key) bool {
-	const (
-		delay    = 15
-		interval = 3
-	)
-	d := inpututil.KeyPressDuration(key)
-	if d == 1 {
-		return true
```

Comment:
I liked, that I could hold down the arrow keys to "scroll" through the entries. Can we get that feature back?

Reply:
It has returned